imperfect-perfect.tex

```diff
@@ -702,6 +700,21 @@ \section{Cohomology of $(\varphi, \Gamma)$-modules}
 (By contrast, the maps $\tilde{\calR}^{[s,r]}_{\QQ_p,A} \to \tilde{\calR}^{[s',r']}_{\QQ_p,A}$ are not completely continuous.)
 \end{remark}
 
+\begin{defn}
+Let $\calR_{K, A}(1)$ denote the object of $\PhiGamma_{K, A}$ corresponding to the cyclotomic character in $\Rep_A(G_K)$.
+There is then a canonical residue map
+\[
+\Res: H^2_{\varphi, \Gamma}(\calR_{K,A}(1)) \to A
+\]
+which when $[K: \QQ_p] < \infty$ corresponds to the residue map from Tate local duality.
```

Comment:
Oh, I should probably assume throughout section 7 that K/Qp is finite, since otherwise it doesn't make sense to talk about the Galois cohomology.

In other news, I moved the definition of the Cartier dual from the end of section 5 to the end of section 1, and took out the explicit definition of the residue map since it's only used implicitly to construct the pairing, and I wasn't planning to write out the definition of that.